src/goto-harness/function_call_harness_generator.cpp

```diff
@@ -217,6 +225,15 @@ void function_call_harness_generatort::implt::generate(
     recursive_initialization_config.variables_that_hold_array_sizes.insert(
       pair.second);
   }
+  recursive_initialization_config.pointers_to_treat_as_cstrings =
+    function_arguments_to_treat_as_cstrings;
+  // std::transform(
+  //   function_argument_to_associated_array_size.begin(),
+  //   function_argument_to_associated_array_size.end(),
+  //   std::inserter(
+  //     recursive_initialization_config.variables_that_hold_array_sizes,
+  //     recursive_initialization_config.variables_that_hold_array_sizes.end()),
+  //   [](const std::pair<irep_idt, irep_idt> &p) { return p.second; });
```

Comment:
Don't add commented code. (and code should be commented using `#if 0`)

Reply:
will be removed, left this in by accident